projects/cm_mcu/ZynqMonTask.c

```diff
@@ -50,16 +52,16 @@
 static void format_data(const uint8_t sensor, const uint16_t data, uint8_t message[4])
 {
   // header and start of sensor (6 bits, sensor[7:2]
-  message[0] = SENSOR_MESSAGE_START_OF_FRAME | ((sensor >> 2) & SENSOR_SIX_BITS);
+  message[0] = SENSOR_MESSAGE_START_OF_FRAME | ((sensor >> 2) & SENSOR_SIX_BITS); // 0b10 ssdddd
   // data frame 1, rest of sensor[1:0] (2 bits) and start of data[15:12] (4 bits)
   message[1] = SENSOR_MESSAGE_DATA_FRAME;
-  message[1] |= ((sensor & 0x3) << 4) | ((data >> 12) & 0xF);
+  message[1] |= ((sensor & 0x3) << 4) | ((data >> 12) & 0xF); // 0b00 dddddd
   // data frame 2, data[11:6] (6 bits)
   message[2] = SENSOR_MESSAGE_DATA_FRAME;
-  message[2] |= (data >> 6) & 0x3F;
+  message[2] |= (data >> 6) & 0x3F; // 0b00 dddddd
   // data frame 3, data[5:0] ( 6 bits )
   message[3] = SENSOR_MESSAGE_DATA_FRAME;
-  message[3] |= data & 0x3F;
+  message[3] |= data & 0x3F; // 0b00 dddddd
 }
```

Comment:
what do these comments mean?

Reply:
it was a note to myself what the encoder does at the beginning of this work. I'll remove it